wemake_python_styleguide/violations/annotations.py

```diff
@@ -87,6 +89,7 @@ class NestedAnnotationsViolation(ASTViolation):
         # Wrong:
         Literal[Literal[Literal[1, 2, 3], "foo"], 5, None]
         Union[Union[int, str], float]
+        NONE_ALIAS = Union[Union[int, str]
```

Comment:
I guess the existing example is good enough. It is about types, not their usage.

Reply:
Removed.